src/pages/DashBoard/Dashboard.tsx

```diff
@@ -1,17 +1,23 @@
-import { Button } from '@material-ui/core';
+import { AppBar, Button } from '@material-ui/core';
 import CircularProgress from '@material-ui/core/CircularProgress';
 import Paper from '@material-ui/core/Paper';
-import { Decrypt, Encrypt } from 'helpers/password.helper';
 
-import React, { Component, useEffect, useState } from 'react';
-import { connect, useSelector } from 'react-redux';
-import { Route, Switch } from 'react-router-dom';
+import React, { useEffect, useState } from 'react';
+import { useDispatch, useSelector } from 'react-redux';
 
+import axios from 'axios-income';
+
+import { LogOut } from 'shared/LogOut/LogOut';
 import Navigation from 'shared/Navigation/Navigation';
+import { reduxIsAuthenticated } from 'store/actions/user';
```

Comment:
Probably we do not want import direct redux function, but action that use this function. Ofcourse - is not fault.